pkg/storage/trb_request_attendee.go

```diff
@@ -115,7 +115,7 @@ func (s *Store) GetTRBRequestAttendeesByTRBRequestID(ctx context.Context, trbReq
 func (s *Store) DeleteTRBRequestAttendee(ctx context.Context, id uuid.UUID) (*models.TRBRequestAttendee, error) {
 	stmt, err := s.db.PrepareNamed(`
 		DELETE FROM trb_request_attendees
-		WHERE id = $1
+		WHERE id = :1
```

Comment:
since you are using named variable, it's expecting the variable to match the property name.
![image](https://user-images.githubusercontent.com/99294987/192884006-5bfbb6c2-4c2c-4c9d-abc6-15a0844d420c.png)